Pidaq/Lib/wind_tunnel/air_properties.py

```diff
@@ -0,0 +1,116 @@
+"""Contains class that defines linear interpolation functions for wind tunnel testing
+
+Classes:
+    LinearInterpolation
+"""
+
+import csv
+import logging
+import sys
+from scipy import interpolate
+
+class LinearInterpolation:
+    """Creates linear interpolation functions for density and dynamic viscosity
+
+    This class defines two functions that are used to linearly interpolate
+    density (kg/m3) and dynamic viscosity (Pa*S) based on temperature (°C)
+    for the purpose of wind tunnel testing.
+    
+    Attributes:
+        densityFunc
+        viscosityFunc
+
+    Methods:
+        interpolateDensity
+        interpolateViscosity
+    """
+
+    def __init__(self, path: str):
+        """
+        Reads a CSV file to define two functions for density and dynamic
+        viscosity that can be used with temperature.  The CSV should have same
+        structure as current one in Drive, where the units are in the second row.
+        
+        Parameters:
+            path - path to a CSV file of the air properties
+
+        Raises:
+            ValueError - A unit (hence the column for a property) was not found
+        """
+        self.logger = logging.getLogger()
+
+        try:
+            open(path) # Open the CSV file
+        except IOError as err:
+            self.logger.error(err)
+            
+        with open(path) as file:
```

Comment:
The init method should probably return following this catch, otherwise it will simply crash on the following call to open() on the next line.  It doesnt really matter what it returns, although you should note what it returns on error in the docstring.

Reply:
Didn't realize this was in the constructor (pre-coffee review rip), but that can be easily alleviated by moving that stuff to a method called init() or similar